internal/dish/controllers/dish.go

```diff
@@ -0,0 +1,154 @@
+package controllers
+
+import (
+	"gin-restapi/internal/common"
+	"gin-restapi/internal/dish/dto"
+	"gin-restapi/internal/dish/services"
+	"gin-restapi/internal/dish/storage"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetDishes(store *storage.SqlStore) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		var paging common.Paging
+
+		if err := c.ShouldBind(&paging); err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"error": err.Error(),
+			})
+
+			return
+		}
+
+		paging.Process()
+
+		sv := services.DishService(store)
+
+		dishes, err := sv.GetDishes(c.Request.Context(), &paging)
+		if err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"error": err.Error(),
+			})
+
+			return
+		}
+
+		c.JSON(http.StatusAccepted, common.SuccessResponse(dishes, paging, nil))
```

Comment:
```suggestion
		c.JSON(http.StatusOk, common.SuccessResponse(dishes, paging, nil))
```
tất cả các request get khi thành công thì trả về status 200 nhé